compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -126,8 +126,8 @@ emitLinkedTuningSpec(ModuleOp module, ArrayRef<NamedSequenceOp> specsToLink) {
   builder.create<transform::YieldOp>(loc, operand);
 
   if (failed(mlir::verify(module))) {
-    module.emitError("verification failed for operation in linked "
-                     "tuning spec");
+    module.emitError("Linked tuning spec failed to verify");
+    return failure();
```

Comment:
I think this should work
```suggestion
    return module.emitError("Linked tuning spec failed to verify");
```